src/containers/pages/IRSLite/MapLite/helpers.ts

```diff
@@ -0,0 +1,269 @@
+import { Dictionary } from '@onaio/utils';
+import mapboxgl, { EventData, LngLat, Map } from 'mapbox-gl';
+import { BLACK, GREY, TASK_GREEN, TASK_ORANGE, TASK_RED, TASK_YELLOW } from '../../../../colors';
+import { STATUS_HEADER } from '../../../../configs/lang';
+import { circleLayerConfig, fillLayerConfig, lineLayerConfig } from '../../../../configs/settings';
+import { FEATURE } from '../../../../constants';
+import { STRUCTURE_LAYER } from '../../../../constants';
+import { GenericJurisdiction } from '../../../../store/ducks/generic/jurisdictions';
+import { StructureFeatureCollection } from '../../../../store/ducks/generic/structures';
+
+/** The default indicator stop */
+export const defaultIndicatorStop = [
+  ['Sprayed', TASK_GREEN],
+  ['Not Sprayed', TASK_RED],
+  ['Partially Sprayed', TASK_ORANGE],
+  ['Not Visited', TASK_YELLOW],
+  ['Not Eligible', BLACK],
+];
+
+/** IRS Indicator stops
+ * These are all the indicator stops for IRS that we know about.
+ */
+export const IRSIndicatorStops: { [key: string]: string[][] } = {
```

Comment:
tiny pick: rename this const to 'IRSLiteIndicatorStops'

Reply:
I just removed this entirely in favor of just  utilizing default:

/** The default indicator stop */
export const defaultIndicatorStop = [
  ['Sprayed', TASK_GREEN],
  ['Not Sprayed', TASK_RED],
  ['Partially Sprayed', TASK_ORANGE],
  ['Not Visited', TASK_YELLOW],
  ['Not Eligible', BLACK],
];

/** IRS Lite Indicator stops
 * These are all the indicator stops for IRS Lite that we know about.
 */
export const IRSLiteIndicatorStops: { [key: string]: string[][] } = {
  zambia2020: defaultIndicatorStop,
};